src/CMakeLists.txt

```diff
@@ -742,7 +741,7 @@ else()
 		endif()
 	endif()
 
-	set(CMAKE_CXX_FLAGS_RELEASE "-DNDEBUG ${RELEASE_WARNING_FLAGS} ${WARNING_FLAGS} ${OTHER_FLAGS} -Wall -pipe -funroll-loops")
+	set(CMAKE_CXX_FLAGS_RELEASE "-DNDEBUG ${RELEASE_WARNING_FLAGS} ${OTHER_FLAGS} -pipe -funroll-loops")
```

Comment:
Just to confirm, it looks like this will no longer use `-Wall` when `WARN_ALL` is false wheras before it always used it. Is that correct?

Reply:
Yes. This is what `WARN_ALL` was supposed to do after all.